index.js

```diff
@@ -169,10 +176,61 @@ module.exports = (robot) => {
     authorize(JSON.parse(clientSecret), checkClaSheet);
   };
 
+  var checkMergeConflicts = async function(context) {
+    var mergeConflictLabel = ['Has Merge Conflicts'];
+    pullRequestsPromiseObj = await context.github.pullRequests.getAll(
+      context.repo());
+
+    arrayOfOpenPullRequests = pullRequestsPromiseObj.data;
+    var hasMergeConflictLabel;
+    for (var indexOfPullRequest in arrayOfOpenPullRequests) {
+      pullRequestNumber = await arrayOfOpenPullRequests[

```

Comment:
will `await` cause the next line's version of `pullRequestNumber` to have a null if there is a nontrivial delay? I would actually suggest wrapping those functions together that will call the following when they get a value.

Reply:
The function is asynchronous meaning it will return a promise. Therefore await can be used here. The await keyword, as the name suggests, pause the flow of execution until the promise is resolved. The next line will only be called when pullRequestNumber gets a value.